app/src/styles.sass

```diff
@@ -44,6 +44,24 @@ input
       background-color: tame($userbuttonbgcolor)
       color: tame($useboxfgcolor)
 
+.formactionscenter
+  padding: $small-space 0
+  display: flex
+  justify-content: center
+  button
+    @include userbox
+    background-color: $userbuttonbgcolor
+    font-size: $fontsize
+    margin-left: $medium-space
+    padding: 0.4em 1em
+    &:hover
+      background-color: color.adjust($maincolor, $lightness: 20%)
+      color: $fgOnMaincolor
+    &[disabled], &[disabled]:hover
+      border-color: tame($maincolor)
+      background-color: tame($userbuttonbgcolor)
+      color: tame($useboxfgcolor)
```

Comment:
Copy-pasting is a plague in computer engineering, because it will force all forthcoming changes to be duplicate. Here you have two classes differing in only one parameter. Your code must reflect clearly that fact. There are more than one way to do it, here are two of them:
- **Login buttons are a special case of standard buttons.** To reflect that, you could make the divs containing login button belonging to both class formactions and formactionscenter. In the CSS, you just add a rule with selector `.formactions.formactioncenter`.
- **Login buttons and standard buttons are two variant of button bars.** To reflect that, you can have all the common parameters in a CSS block with selector `.formactions, .formactionscenter` and then add specific parameters in two other blocks with selectors `.formactions` and `.formactionscenter`.

Reply:
Good morning,
the latest commit contains my two new changements:

according to your request, I have changed the size of the button to be bigger and the "div of submit button" to be the similar size as "mat-form-field" above.
I have reduced the duplicated code in styles.sass and I have chosen the .formactions.login to make the button a special case of standard buttons

Reply:
The div.formations already had the same width as the mat-form-field above. The button is not yet bigger.